content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,250 @@
+2. DNS resolves the destination name to a Gateway address.
+3. The reverse proxy receives the HTTP request on a listener and uses the Host header
+   to match an HTTPRoute.
+4. Optionally, the reverse proxy can perform request header and/or path matching based
+   on match rules of the HTTPRoute.
+5. Optionally, the reverse proxy can modify the request, i.e. add/remove headers, based
+   on filter rules of the HTTPRoute.
+6. Lastly, the reverse proxy forwards the request to one or more objects, i.e. Service,
+   based on rules of the HTTPRoute.
+
+## Conformance
+
+Gateway API covers a broad set of features and is widely implemented. This combination requires
+clear conformance definitions and tests to ensure that the API provides a consistent experience
+wherever it is used.
+
+When considering Gateway API conformance, it's important to understand these concepts:
+
+1. __Release Channels:__ Indicate the stability of a field or resource. The "standard" channel of
+   the API includes fields and resources that have graduated to "beta". The "experimental" channel
+   of the API includes everything in the "standard" channel, along with experimental fields and
+   resources that may still be changed in breaking ways or removed altogether. For more information
+   on this concept, refer to the [versioning][versioning] documentation.
+2. __Support Levels:__ Some implementations of the API may not support every feature. To address this,
+   the API defines _Core_, _Extended_, and _Implementation-specific_ support levels for each feature:
+
+   * __Core__ feature is portable across ALL implementations and is validated by conformance tests. A Core
+     feature is part of the API types schema.
+   * __Extended__ feature is portable but not universally supported across implementations. Implementations
+     that support the feature will have the same behavior and semantics. It is expected that some number of
+     roadmap features will eventually migrate into the _Core_. An _Extended_ feature is part of the API types
+     schema.
+   * __Implementation-specific__ feature is not portable across implementations and is vendor-specific.
+     An _Implementation-specific_ feature will not have API types and schema, except via generic extension
+     points. The implementation is responsible for providing conformance tests to validate these features.
+
+1. __Conformance Tests:__ Gateway API includes a set of conformance tests to validate _Core_ and _Extended_
+   features.
+
+## Getting started
+
+Instead of Gateway API resources being natively implemented by Kubernetes, the specifications
+are defined as [Custom Resources][crds] supported by a wide range of [implementations][implementations].
+You can [install][install] the Gateway API CRDs manually, but this step is typically accomplished by
+your selected implementation. Use one of the following tabs to install a Gateway API implementation:
```

Comment:
```suggestion
You can [install][install] the Gateway API CRDs manually, but a better way is to follow
the installation instructions for your selected implementation.

Use one of the following tabs to install a Gateway API implementation:
```

I've actually become -1 on automatically installing the CRDs, because there are a bunch of weird edge cases that can happen if multiple implementations are present in the cluster and want different versions. (Writing out better rules here is a TODO for me at the moment though). In any case, I recommend using this version here, wherever it ends up living, so that we're not making an implication about what's better.

I agree with the comments below that we should just link to the Gateway API Getting Started page here, and then update that page to make the steps clearer.

Reply:
@youngnick I agree with your -1 after the upstream implementation docs are updated accordingly. In the meantime, I am concerned that an implementation may not properly handle this use case. I have updated this section based on the upstream docs, PTAL.